socorro/external/es/crashstorage.py

```diff
@@ -325,9 +321,8 @@ def _submit_crash_to_elasticsearch(self, crash_document):
         crash_id = crash_document['crash_id']
 
         # Attempt to create the index; it's OK if it already exists.
-        if es_index not in self.indices_cache:
```

Comment:
Nothing updates `indices_cache`, so this code was just hanging out and never had any effect on anything.